apps/librelingo_audios/librelingo_audios/generate_audios.py

```diff
@@ -0,0 +1,88 @@
+from librelingo_utils import audio_id, iterate_phrases
+from pathlib import Path
+import subprocess
+import json
+import random
+
+def generate_audios_for_course(output_path, course_name, course, settings):
```

Comment:
```suggestion
def update_audios_for_course(output_path, course_name, course, settings):
```

since I think it doesn't necessarily always only generate, but potentially also:

- updates
- deletes
- downloads

no? 🤔